File: benchmark/results/java-invocation-script/run-001/b-baseline/output/com/metaengine/demo/identity/services/TokenService.java
package com.metaengine.demo.identity.services;

import com.metaengine.demo.identity.aggregates.User;
import java.util.List;

/** TokenService for the identity domain. */
public class TokenService {

    /** Issue a token for a User. */
    public User create(User input) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** Find a User by id; may return null. */
    public User findById(String id) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** List users up to the given limit. */
    public List<User> list(int limit) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** Delete a user by id. */
    public void delete(String id) {
        throw new UnsupportedOperationException("not implemented");
    }
}

File: benchmark/results/python-multilang/run-004/b-baseline/output/metaengine_demo/identity/services/token_service.py
"""TokenService for the identity domain."""
from typing import Optional

from metaengine_demo.identity.aggregates.user import User


class TokenService:
    """Service issuing and validating tokens."""

    def create(self, input: Optional[User]) -> User:
        """Issue a token for a User."""
        raise NotImplementedError("not implemented")

    def findById(self, id: str) -> Optional[User]:
        """Find token data for a User by id."""
        raise NotImplementedError("not implemented")

    def list(self, limit: float) -> list[User]:
        """List tokenized Users up to limit."""
        raise NotImplementedError("not implemented")

    def delete(self, id: str) -> None:
        """Revoke a token for a User by id."""
        raise NotImplementedError("not implemented")
